packages/cli/src/commands/governance/propose.ts

```diff
@@ -32,6 +36,23 @@ export default class Propose extends BaseCommand {
       .exceedsProposalMinDeposit(deposit)
       .runChecks()
 
+    const builder = new ProposalBuilder(this.kit)
+
+    // BUILD FROM JSON
+    const jsonString = readFileSync(res.flags.jsonTransactions).toString()
+    const jsonTransactions: ProposalTransactionJSON[] = JSON.parse(jsonString)
+    jsonTransactions.forEach((tx) => builder.addJsonTx(tx))
+
+    // BUILD FROM CONTRACTKIT FUNCTIONS
```

Comment:
Remove? Or is this intended to be left as an example?

Reply:
was leaving as an example